apps/storybook.namekit.io/stories/Namekit/utils.ts

```diff
@@ -72,6 +77,11 @@ export const getENSnameFor = (variant: ENSNameVariant) => {
   }
 };
 
+export const MOCKED_DOMAIN_CARD_IS_OWNER_ADDRESS: `0x${string}` =
+  "0x1a199654959140E5c1A2F4135fAA7Ba2748939C6";
+export const MOCKED_DOMAIN_CARD_NON_OWNER_ADDRESS =
+  MOCKED_DOMAIN_CARD_IS_OWNER_ADDRESS.replace("5", "6") as `0x${string}`;
```

Comment:
Better to just hardcode a different address. Please see the more import comment below. The main issue isn't hardcoding addresses. The main issue is a bigger concept. Please see below.

Reply:
Hi @lightwalker-eth! These addresses are not used by the Ui components but only by Storybook stories, in order to replicate the Ui components use cases!